django/contrib/postgres/aggregates/statistics.py

```diff
@@ -0,0 +1,118 @@
+from django.db.models import F, FloatField, IntegerField
+from django.db.models.aggregates import Aggregate
+from django.db.models.expressions import Value
+from django.utils import six
+
+__all__ = [
+    'CovarPop', 'Corr', 'RegrAvgX', 'RegrAvgY', 'RegrCount', 'RegrIntercept',
+    'RegrR2', 'RegrSlope', 'RegrSXX', 'RegrSXY', 'RegrSYY', 'StatFunc',
+]
+
+
+class StatFunc(Aggregate):
+    template = "%(function)s(%(y)s, %(x)s)"
```

Comment:
I commented template and custom `as_sql`, and `tests.postgres_tests.test_aggregates.TestStatisticsAggregate#test_regr_avgx_with_related_obj_and_number_as_argument` failed, because X and Y was in wrong order.
I think we should explicitly see what is going on while constructing sql.
What do you think?